setup.rst

```diff
@@ -52,6 +55,9 @@ executable that needs to be installed on your system only once:
         # ... then, execute the command as
         c:\> cd projects
         c:\projects\> php symfony
+        # in windows 10 with xampp...
+        c:\> cd projects
+        c:\projects\> php.exe symfony
```

Comment:
You are right... I used php (not php.exe) and that's works...
The only problem is that
c:\projects\> php symfony
doesn't create another file. I need use "php symfony" each time that must use symfony.
When I try symfony alone, I have errors:
c:\xampp\htdocs\symfonywebs>symfony
'symfony' n’est pas reconnu en tant que commande interne
ou externe, un programme exécutable ou un fichier de commandes.
the file 'symfony' is the PATH directory and in the directory where I execute that command.
What can be the problem?